plugins/scaffolder-backend-module-rails/src/actions/fetch/rails/index.test.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const mockRailsTemplater = { run: jest.fn() };
+jest.mock('@backstage/plugin-scaffolder-backend', () => ({
+  ...jest.requireActual('@backstage/plugin-scaffolder-backend'),
+  fetchContents: jest.fn(),
+}));
+jest.mock('./railsNewRunner', () => {
+  return {
+    RailsNewRunner: jest.fn().mockImplementation(() => {
+      return mockRailsTemplater;
+    }),
+  };
+});
+
+import {
+  ContainerRunner,
+  getVoidLogger,
+  UrlReader,
+} from '@backstage/backend-common';
+import { ConfigReader } from '@backstage/config';
+import { ScmIntegrations } from '@backstage/integration';
+import mock from 'mock-fs';
```

Comment:
```suggestion
import mockFs from 'mock-fs';
```

this being named just plain "mock" really surprised me farther down in the code :)